rll/test-script.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash -ex
```

Comment:
Noted that test flags set at `#!` vs using `set` from c4e69be429f0175028e7fe11948bd6e014731aa0